pygfx/renderers/wgpu/_wgpurenderer.py

```diff
@@ -99,6 +100,7 @@ def __init__(self, canvas=None, size=None, pixel_ratio=None):
         # and srgb for perseptive color mapping.
         self._canvas_texture = RenderTexture(wgpu.TextureFormat.bgra8unorm_srgb)
         self._depth_texture = RenderTexture(wgpu.TextureFormat.depth32float)
+        self._pick_texture = RenderTexture(wgpu.TextureFormat.rgba32sint)
```

Comment:
Note to self: what about using f32 instead? Many values passed here are floats.

Reply:
Encoding a float by multiplying it is good enough. The other way around is trickier.  Also, encoding multiple small fixed-point values into 1 32 bit int is easier than encoding three values in a float. So leaving this int.